src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbstractAccessControlNameCheck.java

```diff
@@ -140,7 +140,7 @@ public void setApplyToPackage(boolean applyTo) {
     }
 
     /**
-     * Setter to control whether we should apply the check to private members.
+     * Setter to controls whether to apply the check to private members.
```

Comment:
```suggestion
     * Setter to control if check should apply to private members.
```

Please do this for all.

Reply:
done